.github/dependabot.yaml

```diff
@@ -21,6 +21,7 @@ updates:
       - mondoreale
       - juhah
       - kare
+      - daisydomergue
```

Comment:
- Should be with capital D, -> DaisyDomergue 
- Remove juhah, kare and modoreale